packages/md2mld/md2mld.0.1.1/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+name: "md2mld"
+synopsis: "Md files into odoc mld files"
+description: """
+It is a really dumb script, does nothing special.
```

Comment:
It might be worth refining this description slightly to make it a bit more descriptive :-)

```suggestion
md2mld converts a Markdown-format file into the `mld` format used by [odoc](https://github.com/ocaml/odoc) to render HTML documentation or OCaml libraries.  You can use this script to automatically embed a README.md file into API documentation for an OCaml  library.
```

Reply:
I did not notice that dune-release had picked it up 😂

I will also update the description on the repository